internal/generator/fluentd/ingress.go

```diff
@@ -367,7 +295,7 @@ const ConcatLines string = `
 {{define "concatLines" -}}
 <filter kubernetes.**>
   @type concat
-  key log
+  key message
```

Comment:
just for my understanding, this changing related to the optimize data?

Reply:
This does not relate to optimization other then I made modifications to the incoming parsing expression and have removed and modified a number of filter configurations.

Reply:
thanks